op-program/client/program.go

```diff
@@ -37,21 +36,18 @@ func Main(logger log.Logger) {
 }
 
 // RunProgram executes the Program, while attached to an IO based pre-image oracle, to be served by a host.
-func RunProgram(logger log.Logger, bootOracle io.Reader, preimageOracle io.ReadWriter, preimageHinter io.ReadWriter) error {
-	bootReader := NewBootstrapOracleReader(bootOracle)
-	bootInfo, err := bootReader.BootInfo()
-	if err != nil {
-		return fmt.Errorf("failed to read boot info: %w", err)
-	}
-	logger.Debug("Loaded boot info", "bootInfo", bootInfo)
+func RunProgram(logger log.Logger, preimageOracle io.ReadWriter, preimageHinter io.ReadWriter) error {
 
 	pClient := preimage.NewOracleClient(preimageOracle)
 	hClient := preimage.NewHintWriter(preimageHinter)
 	l1PreimageOracle := l1.NewPreimageOracle(pClient, hClient)
 	l2PreimageOracle := l2.NewPreimageOracle(pClient, hClient)
+
+	bootInfo := NewBootstrapClient(pClient).BootInfo()
+	logger.Debug("Program Bootstrapped", "bootInfo", bootInfo)
```

Comment:
This is probably worth making Info level - when the fault proof client runs without the host this would be the only real way to confirm what inputs it is actually running with.

Reply:
Good point. Done.